code_schemes/icraf_s01e02.json

```diff
@@ -3,69 +3,373 @@
+        },
+        {
+        "CodeID": "code-801aed92",
+        "CodeType": "Normal",
+        "DisplayText": "one acre fund",
+        "NumericValue": 31,
+        "StringValue": "one_acre_fund",
+        "VisibleInCoda": true
+        },
+        {
+        "CodeID": "code-9f5fbb3b",
+        "CodeType": "Normal",
+        "DisplayText": "mjengo limited",
+        "NumericValue": 32,
+        "StringValue": "mjengo_limited",
+        "VisibleInCoda": true
+        },
+        {
+        "CodeID": "code-b92543a1",
+        "CodeType": "Normal",
+        "DisplayText": "other",
+        "NumericValue": 33,
+        "StringValue": "other",
+        "VisibleInCoda": true
+        },
+        {
+        "CodeID": "code-343195f7",
+        "CodeType": "Normal",
+        "DisplayText": "asking for specific support",
+        "NumericValue": 34,
+        "StringValue": "asking_for_specific_support",
+        "VisibleInCoda": true
+        },
+        {
+        "CodeID": "code-57c1e449",
+        "CodeType": "Normal",
+        "DisplayText": "complains absence of gov/extension workers",
+        "NumericValue": 35,
+        "StringValue": "complains_absence_of_gov/extension_workers",
+        "VisibleInCoda": true
+        },
+        {
+        "CodeID": "code-74026505",
+        "CodeType": "Normal",
+        "DisplayText": "mentions environmental problem",
+        "NumericValue": 36,
+        "StringValue": "mentions_environmental_problem",
+        "VisibleInCoda": true
+        },
+        {
         "CodeID": "code-NA-f93d3eb7",
         "CodeType": "Control",
         "ControlCode": "NA",
         "DisplayText": "NA (missing)",
         "NumericValue": -10,
         "StringValue": "NA",
         "VisibleInCoda": false
-      },
-      {
+        },
+        {
         "CodeID": "code-NS-2c11b7c9",
         "CodeType": "Control",
         "ControlCode": "NS",
         "DisplayText": "NS (skip)",
         "NumericValue": -20,
         "StringValue": "NS",
         "VisibleInCoda": false
-      },
-      {
+        },
+        {
         "CodeID": "code-NC-42f1d983",
         "CodeType": "Control",
         "ControlCode": "NC",
         "DisplayText": "NC (not coded)",
         "NumericValue": -30,
         "StringValue": "NC",
         "VisibleInCoda": true
-      },
-      {
+        },
+        {
         "CodeID": "code-NR-5e3eee23",
         "CodeType": "Control",
         "ControlCode": "NR",
         "DisplayText": "NR (not reviewed)",
         "NumericValue": -40,
         "StringValue": "NR",
         "VisibleInCoda": false
-      },
-      {
+        },
+        {
         "CodeID": "code-NIC-99631cb8",
         "CodeType": "Control",
         "ControlCode": "NIC",
         "DisplayText": "NIC (not internally consistent)",
         "NumericValue": -50,
         "StringValue": "NIC",
         "VisibleInCoda": false
-      },
-      {
+        },
+        {
+        "StringValue": "Noise_Other_Project",
+        "ControlCode": "NOP",
```

Comment:
I presume this came out of discussion with @lukechurch. Does this code need to be added to Core?

Reply:
Yes. Maybe we can add it to the core @lukechurch ? Its however project specific at the moment.